### PR TITLE
Mech fixes

### PR DESCRIPTION
**Pull request by AlexMorgan3817**
# About The Pull Request
<details>
	<ul>
		<li>Fixed mounted sleeper was dropping dudes from inside to holder, not in world.
		<li>Fixed click cooldown for mechs' pilots
		<li>If u rest in mech with things in hand you will droped it out of mech, not in mech.
		<li>When u enter mech u drop everything from hands on the floor.
		<li>Mechs now can't drink hyperzine
    <li>Cyborgs, pai and AI now can't pilot mechs
	</ul>
</details>

### CODE REVIEW

File: code/modules/mechs/_mech_setup.dm [hunk 1]
```diff
@@ -39,6 +39,8 @@
 #define FRAME_WIRED							1
 #define FRAME_WIRED_ADJUSTED				2
 
+#define STANDART_MECH_MOUNTED_WEAPON_VARS use_external_power = TRUE;restrict_safety = TRUE;twohanded = FALSE;self_recharge = TRUE
```
Comment (MadmanMartian) [inline code/modules/mechs/_mech_setup.dm:42]: STANDARD, not STANDART

File: code/modules/mob/mob_defines.dm [hunk 2]
```diff
@@ -193,4 +193,6 @@ While it would be entirely possible to check the mob's move handlers list for th
 
 	var/datum/stat_holder/stats
 
-	var/mob_classification = 0 //Bitfield. Uses TYPE_XXXX defines in defines/mobs.dm.
\ No newline at end of file
+	var/mob_classification = 0 //Bitfield. Uses TYPE_XXXX defines in defines/mobs.dm.
+
+	var/can_be_feeded = 1 //Can be feeded by reagent_container or other things
```
Comment (MadmanMartian) [inline code/modules/mob/mob_defines.dm:198]: can_be_fed, not can_be_feeded

**Comment by MadmanMartian** [comment]
I'd rather we avoid adding more in-line if statement

File: code/modules/mechs/mech_construction.dm [hunk 3]
```diff
@@ -1,9 +1,11 @@
 /mob/living/exosuit/proc/dismantle()
 
 	playsound(src.loc, 'sound/items/Deconstruct.ogg', 50, 1)
-	var/obj/structure/heavy_vehicle_frame/frame = new(get_turf(src))
-	for(var/hardpoint in hardpoints)
-		remove_system(hardpoint, force = 1)
+	var/obj/structure/heavy_vehicle_frame/frame = new/obj/structure/heavy_vehicle_frame{
+		is_wired = FRAME_WIRED_ADJUSTED;
```
Comment (MadmanMartian) [inline code/modules/mechs/mech_construction.dm:5]: Please don't do it like this. If the is_wired or is_reinforced variable were to be changed, I'm not confident the compiler would catch this.

File: code/modules/mechs/_mech_setup.dm (hunk 1 above)
Comment (AlexMorgan3817) [reply code/modules/mechs/_mech_setup.dm:42]: Spell doesn't matter, but okey, i'll do

File: code/modules/mechs/mech_construction.dm (hunk 3 above)
Comment (AlexMorgan3817) [reply code/modules/mechs/mech_construction.dm:5]: Tell me why compiler doesn't understand his so simple feature as prefabs?

Comment (AlexMorgan3817) [reply code/modules/mechs/mech_construction.dm:5]: This variables is in /obj/structure/heavy_vehicle_frame type, not in structure

Comment (AlexMorgan3817) [reply code/modules/mechs/mech_construction.dm:5]: They isn't setting in /obj/structure/heavy_vehicle_frame/New or Initialize

File: code/modules/mechs/mech_interaction.dm [hunk 4]
```diff
@@ -17,9 +17,16 @@
 		body.MouseDrop_T(dropping, user)
 	else . = ..()
 
+/mob/living/exosuit/setClickCooldown(timeout)
+	/*
+	. = list(world.time + timeout, next_click)
+	for(var/mob/p in pilots) . += p.next_click
+	next_click = max(.)
+	*/
```
Comment (mattroks101) [inline code/modules/mechs/mech_interaction.dm:25]: Unless you plan on using this code, please don't commit commented out code.

File: code/modules/mechs/_mech_setup.dm [hunk 5]
```diff
@@ -39,6 +39,8 @@
 #define FRAME_WIRED							1
 #define FRAME_WIRED_ADJUSTED				2
 
+#define STANDARD_MECH_MOUNTED_WEAPON_VARS use_external_power = TRUE;restrict_safety = TRUE;twohanded = FALSE;self_recharge = TRUE
```
Comment (mattroks101) [inline code/modules/mechs/_mech_setup.dm:42]: This is very messy and severely hurts readability. There is no reason to do this over just setting the vars the normal way as they already  were.

File: code/modules/mechs/mech_construction.dm (hunk 3 above)
Comment (MadmanMartian) [reply code/modules/mechs/mech_construction.dm:5]: Still not happy with it. Change it

Comment (mattroks101) [reply code/modules/mechs/mech_construction.dm:5]: I am seconding this.

Comment (AlexMorgan3817) [reply code/modules/mechs/mech_construction.dm:5]: Okey, so

File: code/modules/mechs/_mech_setup.dm (hunk 5 above)
Comment (AlexMorgan3817) [reply code/modules/mechs/_mech_setup.dm:42]: I think isn't cool to leave many code repetitions
But if u talk about tabulation and <br> in define... 🤔

**Comment by AlexMorgan3817** [comment]
Ready4Merge (?)

**Pull request merged**